Biblioteka_Zadanie9.4/app.py

```diff
@@ -0,0 +1,89 @@
+from flask import Flask, jsonify, abort, make_response, request
+from models import bookstore
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "nininini"
+
+
+@app.errorhandler(404)
+def not_found(error):
+    return make_response(jsonify({"error": "Not found", "status_code": 404}), 404)
+
+
+@app.errorhandler(400)
+def bad_request(error):
+    return make_response(jsonify({"error": "Bad request", "status_code": 400}), 400)
+
+
+@app.route("/api/v1/library/", methods=["GET"])
+def library_list_api_v1():
+    return jsonify(bookstore.all())
+
+
+@app.route("/api/v1/library/", methods=["POST"])
+def create_book():
+    if not request.json or any([
```

Comment:
po co ci to any [ not ...] ?
wiesz co to w ogóle robi?
poczytaj o any: https://www.programiz.com/python-programming/methods/built-in/any